python/GangaGaudi/Lib/Applications/GaudiBase.py

```diff
@@ -190,7 +190,7 @@ def cmt(self, command):
                 env=self.getenv(False),
                 cwd=self.user_release_area)
 
-    def unprepare(self):
+    def unprepare(self, force=False):
```

Comment:
Thanks @mesmith75 It should be there anyway as it is defined in the interface:

[IPrepareApp#L61](https://github.com/ganga-devs/ganga/blob/develop/python/Ganga/GPIDev/Adapters/IPrepareApp.py#L61)

This method looks (from quick glance) like it's doing similar stuff to the base class method so I wonder if we ought not be calling that directly. It also has the `self.decrementShareCounter(self.is_prepared.name)` which *used* to be needed to ensure that the share ref counter properly decremented when you unprepared an app. I have a gut feeling that this was streamlined and that maybe we don't need to make this call directly.

However this PR is still valid as it forces the GaudiBase to conform to the interface.